yarn.lock

```diff
@@ -1297,6 +1297,11 @@
   dependencies:
     prop-types "^15.8.1"
 
+"@hookform/error-message@^2.0.1":
+  version "2.0.1"
+  resolved "https://registry.npmjs.org/@hookform/error-message/-/error-message-2.0.1.tgz#6a37419106e13664ad6a29c9dae699ae6cd276b8"
```

Comment:
I wonder why this defaulted to `registry.npmjs.org` despite the `.yarnrc` file...

Reply:
Yeah not sure why...